### ISSUE TITLE
"Open in Visual Studio Button" not working

### ISSUE THREAD

**Issue by Bomtek9** [desktop/desktop#18236]
### The problem

When I am in Github desktop, I click on "Open in Visual Studio Code" and nothing happens. I have tried to uninstall and reinstall and the error persists. If I make changes in VSCode on a project file, the changes show up on GithubDesktop and I can commit them but the "Open in Visual Studio Code" is not responding.

### Release version

Version 3.3.9 (arm64)

### Operating system

MacOS Sonoma 14.2.1

**Comment by steveward** [comment]
Thanks for the report @Bomtek9. Has VSCode previously worked, and then stopped working (possibly after an update to VSCode or Desktop)? Are there any errors showing when you try to open VSCode? Have you restarted GitHub Desktop?

Can you also open the command line and run the following command:

`defaults read /Applications/Visual\ Studio\ Code.app/Contents/Info CFBundleIdentifier`

Let me know the output of that command. It should be `com.microsoft.VSCode`.

**Comment by Bomtek9** [comment]
Hey, yes it was previously working. Yes there was an update to the mac. I deleted and reloaded both VSCode and GitHub Desktop and still the button is not responsive. I entered the command prompt you provided and it said that it did not have the bundle identifier. So, I corrected it and this is what I received: 

clayduplantis@Clays-MacBook-Air-2 ~ % mdls -name kMDItemCFBundleIdentifier -r "/System/Volumes/Data/Users/clayduplantis/Downloads/Visual Studio Code.app"

com.microsoft.VSCode%   

The button is still not working. Should I delete the apps and redownload them?
